_posts/2024-06-26-performance-improvment-hybrid-query-215.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title:  "Boosting Hybrid Query Performance in OpenSearch 2.15"
+authors:
+  - gaievski
+  - varunudr
```

Comment:
I'm adding Varun as a community member only in this PR, he's working on a new blog post and this would make that other PR work a bit easier. In next commit I'll remove him from the list of authors

Reply:
Thanks! I was planning to ask you if he should be added to the blog.